### DISCUSSION TITLE
Any plan to be compatible with Deno?

### DISCUSSION THREAD

**Discussion by theowenyoung** [handlebars-lang/handlebars.js#1876] [Q&A]
Now I have to use `deno run --allow-env="NODE_DEBUG main.ts` to bypass the deno prompt,

```
Deno requests env access to "NODE_DEBUG", Run again with --allow-env to bypass this prompt.
```

any ideas?

**Comment by jaylinski** [comment]
I tried, but I couldn't make this library work with Deno. Mainly because of the npm-dependency to handlebars-parser.
